instrumentation/instacosmos/containerClient.go

```diff
@@ -0,0 +1,118 @@
+// (c) Copyright IBM Corp. 2023
+
+//go:build go1.18
+// +build go1.18
+
+package instacosmos
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/tracing"
+	"github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos"
+)
+
+// ContainerClient is the interface that wraps the methods of *azcosmos.ContainerClient
+type ContainerClient interface {
+	CreateItem(
+		ctx context.Context,
+		partitionKey azcosmos.PartitionKey,
+		item []byte,
+		o *azcosmos.ItemOptions) (azcosmos.ItemResponse, error)
+}
+
+type instaContainerClient struct {
+	database    string
+	containerID string
+	endpoint    string
+	T           tracing.Tracer
```

Comment:
Is there a reason why `T` is in capital letter, in comparison to the other attributes?
Maybe I find the answer by reading the rest of the code.

Reply:
It doesn't need to be public. I have changed it. Good catch!